.travis.yml

```diff
@@ -6,9 +6,13 @@ go:
 
 env:
   global:
-   - ARCH=$(go env GOARCH)
-   - TAG="ci"
-   - CODECOV_TOKEN="d1e39bea-f800-45de-8266-3e9b85e2594a"
+    - TAG="ci"
+    - CODECOV_TOKEN="d1e39bea-f800-45de-8266-3e9b85e2594a"
+  matrix:
+    - XC_ARCH="amd64"
+      BASEIMAGE="ubuntu:16.04"
+    #- XC_ARCH="arm64"
```

Comment:
If another `arch` is added to the matrix, will a binary be generated for that also.?

Reply:
this was idea. but reverted this change for now. The Cross compile on travis with CGO dependencies needs to be resolved first.